test/test-buffers.js

```diff
@@ -1,10 +1,11 @@
 require(__dirname+'/test-helper');
+var allocUnsafe = require('buffer-alloc-unsafe');
 //http://developer.postgresql.org/pgdocs/postgres/protocol-message-formats.html
 
 var buffers = {};
 buffers.readyForQuery = function() {
   return new BufferList()
-    .add(Buffer('I'))
+    .add(allocUnsafe('I'))
```

Comment:
This should probably use [`Buffer.from()`](https://www.npmjs.com/package/buffer-from). `Buffer.allocUnsafe` is just for allocating a new buffer with a specific length